### PR TITLE
Removing System.out.println and replacing to log.

### PR DESCRIPTION
**Pull request by regisrocha3**
Removing the System.out.println in the code.
And add the conditional to instantiate the default value for the message converter in case that is null

### CODE REVIEW

File: consumer/rabbit-consumer/src/main/java/io/pivotal/java/function/rabbit/consumer/RabbitConsumerConfiguration.java [hunk 1]
```diff
@@ -71,7 +75,7 @@
 	@Bean
 	public MessageHandler amqpChannelAdapter(ConnectionFactory rabbitConnectionFactory)
 			throws Exception {
-		System.out.println("In amqpChannelAdapter - FOOBAR-1");
+		logger.debug("In amqpChannelAdapter");
```
Comment (artembilan) [inline consumer/rabbit-consumer/src/main/java/io/pivotal/java/function/rabbit/consumer/RabbitConsumerConfiguration.java:78]: I would say we don't need this log here at all.
That's kinda not useful information to debug something from `@Bean` method...

File: consumer/rabbit-consumer/src/main/java/io/pivotal/java/function/rabbit/consumer/RabbitConsumerConfiguration.java [hunk 2]
```diff
@@ -131,21 +138,15 @@ public void destroy() throws Exception {
...
+				ObjectProvider<ConnectionNameStrategy> connectionNameStrategy) throws Exception {
+			CachingConnectionFactory cf = super.rabbitConnectionFactory(config, connectionNameStrategy);
+			cf.setConnectionNameStrategy(connectionFactory -> "rabbit.sink.own.connection");
 			cf.afterPropertiesSet();
 			return cf;
 		}
-
 	}
 
 }
```
Comment (artembilan) [inline consumer/rabbit-consumer/src/main/java/io/pivotal/java/function/rabbit/consumer/RabbitConsumerConfiguration.java:152]: Please, add a new line in the end of class.

File: consumer/rabbit-consumer/src/main/java/io/pivotal/java/function/rabbit/consumer/RabbitConsumerConfiguration.java [hunk 3]
```diff
@@ -106,10 +110,13 @@ private ConnectionFactory buildLocalConnectionFactory() throws Exception {
...
 	@Bean
 	public RabbitTemplate rabbitTemplate(ConnectionFactory rabbitConnectionFactory) {
-		System.out.println("IN RABBIT TEMPLATE...");
+		logger.debug("IN RABBIT TEMPLATE...");
 		RabbitTemplate rabbitTemplate = new RabbitTemplate(rabbitConnectionFactory);
 		if (this.messageConverter != null) {
 			rabbitTemplate.setMessageConverter(this.messageConverter);
+		} else {
+			// configure default message converter
+			rabbitTemplate.setMessageConverter(this.jsonConverter());
```
Comment (artembilan) [inline consumer/rabbit-consumer/src/main/java/io/pivotal/java/function/rabbit/consumer/RabbitConsumerConfiguration.java:119]: This is wrong. The JSON converter should be used only when `@ConditionalOnProperty(name = "rabbit.converterBeanName",`.

So, make it unconditionally here is not what the logic expects.

If you worry about a default converter, then look into the `RabbitTemplate`: the `SimpleMessageConverter` is used when we don't provide an explicit.

**Comment by regisrocha3** [comment]
Thank you for reply.
Ok, an Pull Request to only remove these lines really no makes sense.

**Comment by artembilan** [comment]
Well, I don't think so. It is still a change into a code base and it has to be reviewed.

Even if we go to source code over here from GH and try to edit those lines, it will suggest us to open PR.

But if you feel like it is an overhead, we will consider to fix it ourselves.

Thanks for understanding.

**Comment by artembilan** [comment]
Fixed as https://github.com/pivotal/java-functions/commit/ffb31b92f2b32d59c0161133bd560c67363746c9.

Thanks for your effort anyway; looking forward for more contribution!

**Pull request closed without merging**